### DISCUSSION TITLE
What is the daemon command for?

### DISCUSSION THREAD

**Discussion by JyFu-py** [steilerDev/icloud-photos-sync#157] [Q&A] [answered]
Hello,

When I start the photos-sync container the daemon starts automatically with env var 
`- SCHEDULE=* 2 * * *`.

When I use the daemon command, it seems to do the same thing, what is this command for?

Thanks,

**Comment by steilerDev** [comment, marked as answer]
The `daemon` command is the same as not specifying any command.

The docker container - by default - starts with the daemon command. The default value for the schedule is '0 2 * * *' -> See https://steilerdev.github.io/icloud-photos-sync/user-guides/cli/

**Comment by JyFu-py** [comment]
ok i had a doubt, thanks for the clarification
